diff --git a/arch/arm64/boot/dts/qcom/sm8250-oneplus-kebab.dts b/arch/arm64/boot/dts/qcom/sm8250-oneplus-kebab.dts
index 2bb30b439..de2a28285 100644
--- a/arch/arm64/boot/dts/qcom/sm8250-oneplus-kebab.dts
+++ b/arch/arm64/boot/dts/qcom/sm8250-oneplus-kebab.dts
@@ -1916,3 +1916,15 @@ L7P: ldo7 {
 		};
 	};
 };
+
+&camss {
+	status = "okay";
+};
+
+&cci0 {
+	status = "okay";
+};
+
+&cci1 {
+	status = "okay";
+};
