diff --git a/arch/arm64/boot/dts/qcom/sm8250-oneplus-kebab.dts b/arch/arm64/boot/dts/qcom/sm8250-oneplus-kebab.dts
index 777c09bcd..0f575b518 100644
--- a/arch/arm64/boot/dts/qcom/sm8250-oneplus-kebab.dts
+++ b/arch/arm64/boot/dts/qcom/sm8250-oneplus-kebab.dts
@@ -1970,7 +1970,7 @@ camf_imx471: camera@1a {
 		assigned-clocks = <&camcc CAM_CC_MCLK4_CLK>;
 		assigned-clock-rates = <19200000>;
 
-		reset-gpios = <&tlmm 144 GPIO_ACTIVE_LOW>;
+		reset-gpios = <&tlmm 144 GPIO_ACTIVE_HIGH>;
 		pinctrl-0 = <&cam_mclk4_default>;
 		pinctrl-names = "default";
 
